app/django_setup.py

```diff
@@ -13,87 +13,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Initialize Django and its internationalization machinery.
-
-In any file that uses django modules, always import django_setup first.
-
-To localize strings in Python files, import either gettext_lazy or ugettext
-from this module as _, then use _('foo') to mark the strings to be translated.
-Use gettext_lazy for strings that are declared before a language has been
-selected; ugettext for those after (ugettext is safe to use in all Handlers)."""
+"""Initialize Django.
+
+We're currently in between web servers: Person Finder originally ran on webapp2,
+which is no longer being maintained, and we're incrementally moving to Django.
+The move is complicated by the fact that the original implementation uses Django
+for templating and therefore requires some Django configuration. However, Django
+can't be initialized more than once. If we try to treat the webapp2 app (with
+its Django templating setup) and the new Django app totally independently, we'll
+have a problem: whichever one happens to serve first from an instance will work
+fine, and then the next one will crash when it tries to intialize Django a
+second time. So, we want to have a single set of Django settings, intialized by
+the Django app regardless of which app serves first from an instance. The
+webapp2 app imports this, this imports the Django app's entrypoint (wsgi), and
+it initializes Django.
```

Comment:
unclear what "it" refers to in this line

Reply:
Clarified.